numpy/_core/strings.py

```diff
@@ -54,9 +54,13 @@
     "zfill",
 
     # _vec_string - Will gradually become ufuncs as well
-    "mod", "decode", "encode", "upper", "lower", "swapcase", "capitalize",
-    "title", "join", "split", "rsplit", "splitlines", "partition",
-    "rpartition", "translate",
+    "upper", "lower", "swapcase", "capitalize", "title",
+
+    # _vec_string - Will probably not become ufuncs
+    "mod", "decode", "encode", "translate",
```

Comment:
Makes sense, maybe in the future if numpy ever grows support for other encodings.